.github/workflows/ci.yml

```diff
@@ -82,11 +82,35 @@ jobs:
           scope: ${{ matrix.scope }}
           stack-version: ${{ matrix.stack-version }}
 
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      # Temporary fix for "ValueError: invalid mode: 'rU' while trying to load binding.gyp"
+      # See https://github.com/nodejs/node-gyp/issues/2219
+      # This can be removed when "node-gyp" is updated
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version-file: '.nvmrc'
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Run test
+        run: npm run test:unit
```

Comment:
I don't know whether we want this step to run...

Reply:
I don't think so since we run this as part of https://github.com/elastic/apm-agent-rum-js/blob/25143ab2c0a057cc1f6510fcced0a739c5cce792/.ci/docker/node-puppeteer/run-test.sh#L25 via `goal: test`